<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>90</y>
      <width>801</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(233, 255, 93)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="one">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="four">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="seven">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="doubleZero">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eight">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>290</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="two">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zero">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>370</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="five">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nine">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="three">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 6, 10)</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="six">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>210</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 151, 5)</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mult">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 185, 0)</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plus">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 185, 0)</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="divide">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>370</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 185, 0)</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minus">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>210</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 185, 0)</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equals">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>290</y>
        <width>101</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 0)</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="transfer">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>101</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(144, 255, 26)</string>
      </property>
      <property name="text">
       <string>Transer</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>701</width>
        <height>75</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(250,250,230)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextEdit" name="transferText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>631</width>
        <height>75</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>-6</y>
        <width>801</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="notepad_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 143, 246)</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculator">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notepad">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Notepad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="financial">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
